### PR TITLE
power-profiles-daemon: 0.8.1 → 0.10.1

### PR DESCRIPTION
**Pull request by jtojnar**
###### Motivation for this change
https://gitlab.freedesktop.org/hadess/power-profiles-daemon/-/blob/0.10.1/NEWS

The new hold functionality might allow to [crash gnome-control-center](https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/1504) but it is not that concerning.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mvnetbiz** [comment]
It's working normally on my laptop.

File: pkgs/os-specific/linux/power-profiles-daemon/default.nix [hunk 1]
```diff
@@ -43,26 +68,101 @@ stdenv.mkDerivation rec {
...
+    # For finding tests.
+    GI_TYPELIB_PATH_original=$GI_TYPELIB_PATH
+    addToSearchPath GI_TYPELIB_PATH "${testTypelibPath}"
+    GI_TYPELIB_PATH_original=$GI_TYPELIB_PATH
+  '';
+
+  postConfigure = ''
+    GI_TYPELIB_PATH=$GI_TYPELIB_PATH_original
+    unset GI_TYPELIB_PATH_original
+  '';
```
Comment (mvnetbiz) [inline pkgs/os-specific/linux/power-profiles-daemon/default.nix:119]: What is this for? It doesn't make a difference in the output.

Comment (jtojnar) [reply pkgs/os-specific/linux/power-profiles-daemon/default.nix]: Thanks, fixed.

Comment (mvnetbiz) [reply pkgs/os-specific/linux/power-profiles-daemon/default.nix]: It looked like it was supposed to stop extra dependncies from getting in the output?

Comment (mvnetbiz) [reply pkgs/os-specific/linux/power-profiles-daemon/default.nix]: Oh that is what it does, just export was missing. I wasn't sure why it was not working.

Comment (jtojnar) [reply pkgs/os-specific/linux/power-profiles-daemon/default.nix]: The issue was actually that I was overwriting GI_TYPELIB_PATH_original after changing it.

**Review by mvnetbiz — APPROVED** [review]

**Pull request merged**